packages/builder/src/utils/graphql.ts

```diff
@@ -19,21 +17,21 @@ export enum ChainId {
  */
 const getGraphQLEndpoint = (
   chainId: number,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   reactEnv?: any // ProcessEnv
```

Comment:
why don't we just delete the variable here and where it's called?

Reply:
It was being used in graphqlFetch function as well & I was not sure if removing it will cause issues in other places or lead to failing tests etc, so left it there for now.

Reply:
the parameter is optional, and now we're not even using it, so not sure what kind of issues removing it will cause, did you check? removing it should be quite easy and will save us a headache in the future, as now we have a parameter in the function signature but it's not use, the caller will not expect this unless they read the code

Reply:
I agree let's remove it, and with a quick search we can check all the calls to the function and remove the param from the calls, since it's not even used here.